janus_core/calculations/eos.py

```diff
@@ -0,0 +1,117 @@
+"""Equation of State."""
+
+from typing import Any, Optional
+
+from ase import Atoms
+from ase.eos import EquationOfState
+from ase.io import read
+from ase.io.trajectory import Trajectory
+from ase.units import kJ
+import numpy as np
+
+from janus_core.calculations.geom_opt import optimize
+from janus_core.helpers.janus_types import PathLike
+from janus_core.helpers.log import config_logger
+from janus_core.helpers.utils import none_to_dict
+
+
+def calc_eos(  # pylint: disable=too-many-locals
+    struct: Atoms,
+    struct_name: Optional[str] = None,
+    min_lattice: float = 0.95,
+    max_lattice: float = 1.05,
+    n_lattice: int = 5,
```

Comment:
```suggestion
    n_lattice: int = 11,
```

Reply:
shall give us 0.95, 096,.... 1.05

Reply:
Not with ` ** (1 / 3)` added to the linspace